appveyor.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.4.{build}
```

Comment:
Is version here used for anything? Or is it just overwritten by the Cake script? I'm probably missing something. 😄